Makefile.pre.in

```diff
@@ -893,6 +895,16 @@ sharedmods: $(SHAREDMODS) pybuilddir.txt
 checksharedmods: sharedmods $(PYTHON_FOR_BUILD_DEPS) $(BUILDPYTHON)
 	@$(RUNSHARED) $(PYTHON_FOR_BUILD) $(srcdir)/Tools/scripts/check_extension_modules.py
 
+rundsymutil: sharedmods $(PYTHON_FOR_BUILD_DEPS) $(BUILDPYTHON)
+	@if [ ! -z $(DSYMUTIL) ] ; then \
```

Comment:
Maybe add an echo statement here to show that dsymutil is used? Especially because the rest of the build is verbose.